src/migration/MigrationExecutor.ts

```diff
@@ -172,14 +192,13 @@ export class MigrationExecutor {
     async executePendingMigrations(): Promise<Migration[]> {
         const queryRunner =
             this.queryRunner || this.connection.createQueryRunner()
-        // create migrations table if its not created yet
+        // create migrations table if it's not created yet
         await this.createMigrationsTableIfNotExist(queryRunner)
 
-        // create the typeorm_metadata table if necessary
+        // create the typeorm_metadata table if it's not created yet
         const schemaBuilder = this.connection.driver.createSchemaBuilder()
-
         if (InstanceChecker.isRdbmsSchemaBuilder(schemaBuilder)) {
-            await schemaBuilder.createMetadataTableIfNecessary(queryRunner)
+            await schemaBuilder.createTypeormMetadataTable(queryRunner)
```

Comment:
is there any issues with `createMetadataTableIfNecessary()` method? Metadata table required only for views and generated columns.